tests/utils/mock_server.py

```diff
@@ -665,7 +688,19 @@ def storage():
         # make sure to read the data
         request.get_data()
         if file == "wandb_manifest.json":
-            if (
+            if _id == "bb8043da7d78ff168a695cff097897d2":
```

Comment:
I'm guessing this will always be the digest from an artifact with an empty? And the check below is for the empty table?

Neat! I was trying add a test for the `wait()` in my earlier PR but couldn't get the mock server to co-operate. Lesson noted for the future!

Reply:
yeah, this is the digest for an artifact containing an empty table, then the item below is that empty table. However, the digest for the table is calculated as a derivative of the provided digest `"3aaaaaaaaaaaaaaaaaaaaa=="`.  This mock server is a pain. I have been wanting to make a clean seed spec so we can get away from these fragile switches. For now it will work tho (: